packages/cursorless-engine/src/languages/python.ts

```diff
@@ -97,10 +67,6 @@ const nodeMatchers: Partial<
   functionCall: "call",
   functionCallee: "call[function]",
   comment: "comment",
-  class: "decorated_definition?.class_definition",
-  className: "class_definition[name]",
```

Comment:
fixed a bug along the way; now you can say "class name" from within the decorator of the class